go/kbfs/libkbfs/tlf_journal_test.go

```diff
@@ -1513,15 +1519,11 @@ func testTLFJournalConvertWhileFlushing(t *testing.T, ver kbfsmd.MetadataVer) {
 	_, _, unpauseBlockPutCh, errCh, blocksLeftAfterFlush, mdsLeftAfterFlush :=
 		testTLFJournalPauseBlocksAndConvertBranch(t, ctx, tlfJournal, config)
 
-	for i := 0; i < 2; i++ {
-		select {
-		// Now finish the block put, and let the flush finish.  We should
-		// be on a local squash branch now.
-		case unpauseBlockPutCh <- struct{}{}:
-		case err := <-errCh:
-			require.NoError(t, err)
-		}
-	}
+	// Now finish the block put, and let the flush finish.  We
+	// shouldn't be on a branch anymore.
```

Comment:
We should be on a branch -- the local squash branch.  In fact there's a require just below that checks that.  Same thing below.